pkg/rescaffold/migrate.go

```diff
@@ -172,6 +176,41 @@ func migrateGrafanaPlugin(store store.Store, src, des string) error {
 	return kubebuilderGrafanaEdit()
 }
 
+func migrateDeployImagePlugin(store store.Store) error {
+	log.Printf("beam2")
+	deployImagePlugin := v1alpha1.PluginConfig{}
+	err := store.Config().DecodePluginConfig("deploy-image.go.kubebuilder.io/v1-alpha", deployImagePlugin)
+	// If the deploy-image plugin is not found, we don't need to migrate
+	if err != nil {
+		if errors.As(err, &config.PluginKeyNotFoundError{}) {
+			log.Printf("deploy-image plugin is not found, skip the migration")
+			return nil
+		}
+		return fmt.Errorf("failed to decode deploy-image plugin config %v", err)
+	}
+	// Here it is always empty, I don't know why
+	// if len(deployImagePlugin.Resources) == 0 {
+	// 	log.Printf("empty deploy-image plugin config, skip the migration")
+	// }
```

Comment:
When DecodePluginConfig, the deployImagePlugin is always empty. I don't know what happened with it. :(